File: app/src/main/java/de/bitb/astroskop/ui/main/IMainView.java
package de.bitb.astroskop.ui.main;


import de.bitb.astroskop.ui.base.IBaseView;

public interface IMainView extends IBaseView {
    void navigateToHomeScreen();

    void navigateToCircumstanceScreen();

    void navigateToScreen(int tabId);
}

File: app/src/main/java/de/bitb/astroskop/ui/main/IMainView.kt
package de.bitb.astroskop.ui.main


import de.bitb.astroskop.ui.base.IBaseView

interface IMainView : IBaseView {
    fun navigateToHomeScreen()

    fun navigateToCircumstanceScreen()

    fun navigateToScreen(tabId: Int)
}
